plugins/plugins.gradle.kts

```diff
@@ -3,6 +3,10 @@ import org.gradle.jvm.tasks.Jar
 
 subprojects {
     afterEvaluate {
+        /** The name of the plugin. This is, by convention, entirely lower-case. */
+        val pluginName: String = project.name
+        /** The plugin name as a proper noun, with the first character capitalized */
+        val proper = Character.toUpperCase(pluginName[0]) + pluginName.substring(1)
```

Comment:
Can we do something smarter here so it follows the name of the plugin rather than blindly capitalizing the first char?

Reply:
That would be ideal. Project names are always lowercase eg `networktables`, `powerup`, `cameraserver` and cannot be reassigned. I suppose a property might be a good enough solution, even though it's not very enforceable